src/sidemenu/PinnedMenu.tsx

```diff
@@ -33,22 +33,41 @@ const PinnedMenuItem: React.FC<
   const { item, current, menuOptions, label } = props as PinnedMenuItemProps;
   const { loading, loadingAccentColor } = props as LoadingPinnedMenuItemProps;
 
+  const getNotificationlabel = (item: BoardType | CircleButtonProps, label?: string) => {
+    if (!label) {return ""}
+    if ("slug" in item) {
+      if (item.muted) {return `${label} muted`;}
+      if (!item.updates) {return label;}
+      if (item.outdated) {return `${label} has updates`;}
+      return `${label} has new updates`;
+    } else if (item.withNotification) {
+      return `${label} has new updates`;
+    } else {
+      return label;
+    }
+  };
+  
   return (
     <div
       className={classNames("pinned-item", {
         button: item && !("slug" in item),
       })}
     >
       {loading ? (
-        <LoadingPlaceholder
-          accentColor={loadingAccentColor || DefaultTheme.DEFAULT_ACCENT_COLOR}
-          height="50px"
-        />
+        <div aria-label="loading pinned item placeholder">
```

Comment:
"loading placeholder for pinned item" or "pinned item loading placeholder"

Reply:
(Unless you have strong opinions. Took me a while to parse it as it was.)

Reply:
Oh, for this there is actually a comment later about potentially changing this to "pinned items are loading"

Reply:
See my comment in reply to the later comment